lib/rspec/matchers.rb

```diff
@@ -646,6 +646,7 @@ def include(*expected)
     alias_matcher :a_string_including,     :include
     alias_matcher :a_hash_including,       :include
     alias_matcher :including,              :include
+    alias_matcher :have_a_member,          :include
```

Comment:
I think this one needs to be "highlighted". Is there any other good suggestion to make `it { is_expected.to include(be_an(Integer)) }` sound "good"? Maybe `it { is_expected.to have_a_member(be_an(Integer)) }`?